folderAWG/Interoperability Precis.md

```diff
@@ -0,0 +1,128 @@
+3. Switching between voice agents. To fulfill user needs, a voice agent may need to switch the user to a second voice agent—and then back again. For instance, an experienced 
+traveler may wish to switch back-and-forth between an airline-specific agent, a hotel chain-specific agent, and the agents of restaurants. 
+4. Processing implicit requests. We define “implicit” as a request that searches across voice agent services rather than seeking specific, named voice agent destination. This will rely upon an arbitration service to select (or recommend) a voice agent from many. 
+5. Sharing data and context between voice agents. This enhancement advances #3, above, and enables agents to manage the sharing of data, context, and controls. It also will enable agents to call upon the universe of conversational capabilities, regardless of platform parentage. 
+6. Generating and operating a private agent persona. A persona refers to the voice and characteristics presented by a conversational agent. Such agents have a distinctive sound and ambiance that leads to branding, the promotion of a particular enterprise by means of advertising, and distinctive design. This enhancement will enable an enterprise voice agent to fully express a specific brand tone, and enable capabilities to adapt to or adopt the persona of the enterprise agent. 
+Other possible enhancements include: 
+● discovery—locating an agent; 
+● planning—developing a sequence of voice agents and capabilities that perform a complex task; and, 
+● identification and authentication of both the user and desired conversational agent, to allow a secure connection. 
+These and other potential enhancements are currently outside of the scope of the Open Voice Network Voice Agent Interoperability initiative. 
+
+### VII. How You Can Accelerate the Open Voice Network Agent-to-Agent Initiative 
+To accelerate the Open Voice Network agent-to-agent interoperability initiative, please consider helping to shape the future as you: 
+● Invest your expertise in the Open Voice Network interoperability Initiative: ○ Participate in weekly debates, discussions, and reviews. Meetings are virtual, recorded, no more than 60 minutes, and, in keeping with OVON culture, they begin and end on time. 
+○ Review and contribute to the work as your schedule permits. Documents (including this one) can be found in the Open Voice Network GitHub repository. Should you see a reason for revision, simply open a pull request. And, you can keep up with the weekly debates and discussions by reviewing the meeting recordings. They’re stored in the Open Voice Network Google Workspace, accessible by members and by requesting permission. 
```

Comment:
A proper way to say it: "...can be found in the Open Voice Network GitHub organization in the docs repository"